src/Backend/Factory.php

```diff
@@ -56,7 +56,11 @@ public function buildRedisCache($options)
         }
 
         $redis = new \Redis();
-        $redis->connect($options['host'], $options['port'], $timeout);
+        if (empty($options['unix_socket'])) {
+            $redis->pconnect($options['host'], $options['port'], $timeout);
```

Comment:
Is the change to pconnect really needed? As far as I can see this would be a persistent connection